python/ray/data/tests/test_huggingface.py

```diff
@@ -8,7 +8,9 @@
 
 
 def test_from_huggingface(ray_start_regular_shared):
-    data = datasets.load_dataset("tweet_eval", "emotion")
+    data = datasets.load_dataset(
+        "tweet_eval", "emotion", download_mode="force_redownload"
```

Comment:
does the test work without `download_mode="force_redownload"` now? if so can we drop it?